packages/clarity-js/src/data/index.ts

```diff
@@ -47,4 +48,5 @@ export function compute(): void {
     summary.compute();
     limit.compute();
     extract.compute();
+    consent.compute();
```

Comment:
encode consent every time you get a value instead of doing this at compute - you might miss intermediate states.